src/pke/lib/scheme/bgvrns/bgvrns-parametergeneration.cpp

```diff
@@ -194,6 +194,71 @@ std::pair<std::vector<NativeInteger>, uint32_t> ParameterGenerationBGVRNS::compu
 
 }
 
+void ParameterGenerationBGVRNS::InitializeFloddingDgg(
+  std::shared_ptr<CryptoParametersBase<DCRTPoly>> cryptoParams, 
+  usint numPrimes,
+  enum KeySwitchTechnique ksTech,
+  enum ProxyReEncryptionMode PREMode) const {
+  
+  const auto cryptoParamsBGVRNS =
+      std::static_pointer_cast<CryptoParametersBGVRNS>(cryptoParams);
+
+  //compute the flooding distribution parameter based on the security mode for pre
+  double stat_sec=30; //statistical security parameter for noise flooding
```

Comment:
no "magic" numbers inside functions and without a name.

Reply:
added a constant STAT_SEC in rlwe-cryptoparameters.h

Reply:
don't mark it fixed unless it is in the branch